Ubs.java

gabrielafg opened the pull request:
Os nomes alguns métodos e da classe foram alterados para deixar seu objetivo mais evidente; criação de variáveis boolean auxiliares nos métodos boolean para facilitar a leitura; uso da classe Calendar do java para puxar o valor do ano direto do sistema, evitando a necessidade de alteração anual no código;

gabrielafg committed:
Update Ubs.java

Os nomes alguns métodos e da classe foram alterados para deixar seu objetivo mais evidente; criação de variáveis boolean auxiliares nos métodos boolean para facilitar a leitura; uso da classe Calendar do java para puxar o valor do ano direto do sistema, evitando a necessidade de alteração anual no código;

mariliarosa4 commented:
Baseado nos conceitos de código limpo, as variáveis estão bem nomeadas, os métodos/funções organizadas e reduzidas.  Seria legal apenas ter uma breve descrição do que essa classe faz, como por exemplo "Código para validação dos dados de entrada em um sistema de Unidade Básica de Saúde".

gabrielafg committed:
Update Ubs.java

gabrielafg committed:
Update Ubs.java

Acrescentou-se uma linha de comentário no início do código para explicar o objetivo da classe.

gabrielafg commented:
De acordo com a sugestão, uma linha de comentário no início do código para explicar o objetivo da classe foi acrescentada.
